app/auto-eligibility/register-your-interest/process-eligible-sbi.js

```diff
@@ -10,9 +12,55 @@ const processEligibleSbi = async (customer) => {
   if (selectYourBusiness.enabled) {
     await customerDbTable.updateWaitingUpdatedAt(customer.sbi, customer.crn)
     await emailNotifier.sendWaitingListEmail(customer.businessEmail)
+    await raiseEvent({
+      name: 'auto-eligibility:incoming-register-your-interest:recognised_as_eligible',
+      properties: {
+        id: `${customer.sbi}_${customer.crn}`,
+        sbi: customer.sbi,
+        cph: 'n/a',
+        checkpoint: appInsightsConfig.appInsightsCloudRole,
+        status: 'SUCCESS',
+        action: {
+          type: 'put_on_the_waiting_list',
+          message: 'The customer has been put on the waiting list',
+          data: {
+            customer: {
+              sbi: customer.sbi,
+              crn: customer.crn,
+              businessEmail: customer.businessEmail
+            }
+          },
+          raisedOn: new Date(),
+          raisedBy: 'auto-eligibility:incoming-register-your-interest'
+        }
+      }
+    })
   } else {
     if (customer.businessEmailHasMultipleDistinctSbi) {
       console.log(`${new Date().toISOString()} The customer's business email has multiple distinct SBI`)
+      await raiseEvent({
+        name: 'auto-eligibility:incoming-register-your-interest:rejected_due_to_multiple_sbi',

```

Comment:
maybe we can make an enum/constant with all the event names?

Reply:
done